commands/registry.go

```diff
@@ -259,9 +329,123 @@ func RunRegistryLogout(c *CmdConfig) error {
 	return cmd.Run()
 }
 
+// Repository Run Commands
+
+// RunListRepositories lists repositories for the registry
+func RunListRepositories(c *CmdConfig) error {
+	registry, err := c.Registry().Get()
+	if err != nil {
+		return fmt.Errorf("failed to get registry: %w", err)
+	}
+
+	repositories, err := c.Registry().ListRepositories(registry.Name)
+	if err != nil {
+		return err
+	}
+
+	return displayRepositories(c, repositories...)
+}
+
+// RunListRepositoryTags lists tags for the repository in a registry
+func RunListRepositoryTags(c *CmdConfig) error {
+	if len(c.Args) != 1 {
+		return doctl.NewMissingArgsErr(c.NS)
+	}
+
+	registry, err := c.Registry().Get()
+	if err != nil {
+		return fmt.Errorf("failed to get registry: %w", err)
+	}
+
+	tags, err := c.Registry().ListRepositoryTags(registry.Name, c.Args[0])
+	if err != nil {
+		return err
+	}
+
+	return displayRepositoryTags(c, tags...)
+}
+
+// RunRepositoryDeleteTag deletes one or more repository tags
+func RunRepositoryDeleteTag(c *CmdConfig) error {
+	force, err := c.Doit.GetBool(c.NS, doctl.ArgForce)
+	if err != nil {
+		return err
+	}
+
+	if len(c.Args) < 2 {
+		return doctl.NewMissingArgsErr(c.NS)
+	}
+
+	registry, err := c.Registry().Get()
+	if err != nil {
+		return fmt.Errorf("failed to get registry: %w", err)
+	}
+
+	repository := c.Args[0]
+	tags := c.Args[1:]
+
+	if !force && AskForConfirm(fmt.Sprintf("delete %d repository tag(s)", len(tags))) != nil {
+		return fmt.Errorf("operation aborted")
+	}
+
+	for _, tag := range tags {
+		if err := c.Registry().DeleteTag(registry.Name, repository, tag); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+// RunRepositoryDeleteManifest deletes one or more repository manifests by digest
+func RunRepositoryDeleteManifest(c *CmdConfig) error {
+	force, err := c.Doit.GetBool(c.NS, doctl.ArgForce)
+	if err != nil {
+		return err
+	}
+
+	if len(c.Args) < 2 {
+		return doctl.NewMissingArgsErr(c.NS)
+	}
+
+	registry, err := c.Registry().Get()
+	if err != nil {
+		return fmt.Errorf("failed to get registry: %w", err)
+	}
+
+	repository := c.Args[0]
+	digests := c.Args[1:]
+
+	if !force && AskForConfirm(fmt.Sprintf("delete %d repository manifest(s) by digest", len(digests))) != nil {
```

Comment:
At risk of making the confirmation too wordy, it would be nice to include a warning here that any associated tags will also be deleted.